### PR TITLE
s2s Individual time logging + ss flag addition

### PR DESCRIPTION
**Pull request by pm-dattaprasad-mundada**

### CODE REVIEW

File: src_new/adapters/prebid.js [hunk 1]
```diff
@@ -38,7 +38,14 @@ function transformPBBidToOWBid(bid, kgpv){
 	theBid.setAdUrl(bid.adUrl || "");
 	theBid.setWidth(bid.width);
 	theBid.setHeight(bid.height);
-	theBid.setReceivedTime(bid.responseTimestamp);
+
+	if (theBid.getServerSideStatus() === 0) {
```
Comment (pm-dattaprasad-mundada) [inline src_new/adapters/prebid.js]: Remove if else from here,

theBid.setReceivedTime(bid.responseTimestamp);
theBid.setServerSideResponseTime(bid.serverSideResponseTime);

Comment (pm-dattaprasad-mundada) [reply src_new/adapters/prebid.js]: Only above two statement is required, bid.ttl is something not related to responseTime

File: src_new/bid.js [hunk 2]
```diff
@@ -12,17 +12,35 @@ function Bid(adapterID, kgpv){
...
+
+Bid.prototype.getServerSideResponseTime = function () {
+	return this.serverSideResponseTime;
+};
+
+Bid.prototype.getServerSideStatus = function () {
+	return this.isServerSide;
+};
+
+Bid.prototype.setServerSideStatus = function (isServerSide) {
```
Comment (PubMatic-OpenWrap) [inline src_new/bid.js]: this function is not being used, we can comment it for now

File: src_new/bidManager.js [hunk 3]
```diff
@@ -389,9 +389,9 @@ function analyticalPixelCallback(slotID, bmEntry, impressionIDMap) { // TDD, i/o
                 return;
             }
 
-						if (adapterID === "pubmaticServer") {
-								return;
-						}
+						// if (adapterID === "pubmaticServer") {
```
Comment (PubMatic-OpenWrap) [inline src_new/bidManager.js]: I think block should not be commented.
This block was added to not pass data for pubmaticServer adapter into logger as it is not actually an adapter but a container of server-side adapters.

**Pull request merged**